pkg/bot/discord.go

```diff
@@ -153,21 +153,16 @@ func (b *Discord) SendEvent(_ context.Context, event event.Event, eventSources [
 	return errs.ErrorOrNil()
 }
 
-// SendGenericMessage sends interactive message to selected Discord channels.
+// SendMessage sends interactive message to selected Discord channels.
 // Context is not supported by client: See https://github.com/bwmarrin/discordgo/issues/752.
-func (b *Discord) SendGenericMessage(_ context.Context, genericMsg interactive.GenericMessage, sourceBindings []string) error {
-	msg := genericMsg.ForBot(b.BotName())
-
+func (b *Discord) SendMessage(_ context.Context, msg interactive.CoreMessage, sourceBindings []string) error {
 	errs := multierror.New()
 	for _, channelID := range b.getChannelsToNotify(sourceBindings) {
-		b.log.Debugf("Sending message to channel %q: %+v", channelID, msg)
```

Comment:
Can we retain some of the debug logging?

Reply:
it is still logged in the `send` method, so I just removed it to don't have duplicates :)